sms/src/main/java/org/motechproject/sms/http/GenericResponseHandler.java

```diff
@@ -39,7 +38,7 @@ public void handle(OutgoingSms sms, String response, Header[] headers) {
                     sms.getRecipients().toString()));
             for (String recipient : sms.getRecipients()) {
                 getAuditRecords().add(new SmsRecord(getConfig().getName(), OUTBOUND, recipient, sms.getMessage(), now(),
-                        DeliveryStatus.DISPATCHED, null, sms.getMotechId(), providerMessageId, null));
+                        getTemplateOutgoingResponse().getSuccessStatus(), null, sms.getMotechId(), providerMessageId, null));
```

Comment:
This is not the correct value, success status is a pattern against the response code is matched. So it takes values like '200' or '20*', it's not something we would store as the status.